tests/unit_tests/test_wave_06.py

```diff
@@ -110,16 +109,21 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That the results is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result 
```

Comment:
Instead of asserting that result is a truthy value, we want to be explicit that we're checking to see if result is actually equal to `True`. The README tells us that this function should return `True` so checking this will ensure that your function is returning the correct value and not just some truthy value (a list with an element in is it truthy, an integer that isn't 0 is truthy, etc).

While we would normally write in Python `if result:`

here we want to be more explicit

Reply:
✅ Fixed (will push later!)